x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_list.ts

```diff
@@ -64,8 +66,9 @@ export function useFetchSloList({
           // ignore error
         }
       },
-      refetchOnWindowFocus: false,
       keepPreviousData: true,
+      refetchOnWindowFocus: false,
+      refetchInterval,
```

Comment:
Should we also refresh the historical data and alert calls?

Reply:
Yes, you're right